rubin_sim/scheduler/example/example_scheduler.py

```diff
@@ -14,24 +16,191 @@
     LongGapSurvey,
     generate_ddf_scheduled_obs,
 )
-import rubin_sim.scheduler.detailers as detailers
 from rubin_sim.scheduler import sim_runner
-from astropy.coordinates import SkyCoord, get_sun
-from astropy.time import Time
-from astropy import units as u
-from rubin_sim.utils import _hpid2_ra_dec
-import warnings
+import rubin_sim.scheduler.detailers as detailers
 import sys
 import subprocess
-import rubin_sim
 import os
 import argparse
-from rubin_sim.scheduler.model_observatory import ModelObservatory
+from astropy.coordinates import SkyCoord
+from astropy import units as u
+from rubin_sim.utils import _hpid2_ra_dec
+import rubin_sim
+import warnings
 
+# So things don't fail on hyak
+from astropy.utils import iers
+
+iers.conf.auto_download = False
 
 __all__ = ["example_scheduler"]
 
 
+def standard_bf(
+    nside,
+    filtername="g",
+    filtername2="i",
+    m5_weight=6.0,
+    footprint_weight=1.5,
+    slewtime_weight=3.0,
+    stayfilter_weight=3.0,
+    template_weight=12.0,
+    u_template_weight=50.0,
+    g_template_weight=50.0,
+    footprints=None,
+    n_obs_template=None,
+    season=300.0,
+    season_start_hour=-4.0,
+    season_end_hour=2.0,
+    moon_distance=30.0,
+):
+    """Generate the standard basis functions that are shared by blob surveys
+
+    Parameters
+    ----------
+    XXX
+
+
+    Returns
+    -------
+    list of tuples pairs (basis function, weight) that is
```

Comment:
If we're following DM's lead, we should follow the way that define types, [here](https://developer.lsst.io/python/numpydoc.html#py-docstring-returns) and [here](https://developer.lsst.io/python/numpydoc.html#py-docstring-parameter-types)